packages/connect-examples/exampleV2/DrawIO.md

```
@@ -0,0 +1,132 @@
+# DrawIO 专业设计助手
+
+## 角色定位
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)